internal/provider/serverless_export_resource.go

```diff
@@ -0,0 +1,761 @@
+								MarkdownDescription: "Separator of each value in CSV files.",
+								Optional:            true,
+								Computed:            true,
+								PlanModifiers: []planmodifier.String{
+									stringplanmodifier.UseStateForUnknown(),
+								},
+							},
+							"delimiter": schema.StringAttribute{
+								MarkdownDescription: "Delimiter of string type variables in CSV files.",
+								Optional:            true,
+								Computed:            true,
+								PlanModifiers: []planmodifier.String{
+									stringplanmodifier.UseStateForUnknown(),
+								},
+							},
+							"null_value": schema.StringAttribute{
+								MarkdownDescription: "Representation of null values in CSV files.",
+								Optional:            true,
+								Computed:            true,
+								PlanModifiers: []planmodifier.String{
+									stringplanmodifier.UseStateForUnknown(),
+								},
+							},
+							"skip_header": schema.BoolAttribute{
+								MarkdownDescription: "Export CSV files of the tables without header.",
+								Optional:            true,
+								Computed:            true,
+								PlanModifiers: []planmodifier.Bool{
+									boolplanmodifier.UseStateForUnknown(),
+								},
+							},
+						},
+					},
+					"parquet_format": schema.SingleNestedAttribute{
+						MarkdownDescription: "The format of the parquet.",
+						Optional:            true,
+						Computed:            true,
+						PlanModifiers: []planmodifier.Object{
+							objectplanmodifier.UseStateForUnknown(),
+						},
+						Attributes: map[string]schema.Attribute{
+							"compression": schema.StringAttribute{
+								MarkdownDescription: "The compression of the parquet.",
+								Optional:            true,
+								Computed:            true,
+								PlanModifiers: []planmodifier.String{
+									stringplanmodifier.UseStateForUnknown(),
+								},
+							},
+						},
+					},
+				},
+			},
+			"target": schema.SingleNestedAttribute{
+				MarkdownDescription: "The target of the export.",
+				Optional:            true,
+				Computed:            true,
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"type": schema.StringAttribute{
+						MarkdownDescription: "The exported file type.",
+						Optional:            true,
+						Computed:            true,
+						PlanModifiers: []planmodifier.String{
+							stringplanmodifier.UseStateForUnknown(),
+						},
+					},
+					"s3": schema.SingleNestedAttribute{
+						MarkdownDescription: "S3 target.",
+						Optional:            true,
+						Computed:            true,
+						PlanModifiers: []planmodifier.Object{
+							objectplanmodifier.UseStateForUnknown(),
+						},
+						Attributes: map[string]schema.Attribute{
+							"uri": schema.StringAttribute{
+								MarkdownDescription: "The URI of the s3 folder.",
+								Optional:            true,
+								Computed:            true,
```

Comment:
why s3 attributes have both optional and computed and azure/gcs not

Reply:
I'm not sure about the return values of these target types, I need help from @shiyuhang0